### DISCUSSION TITLE
CrazyBolt 1.1 has wrong battery voltage and cannot fly

### DISCUSSION THREAD

**Discussion by mfxjx000205** [bitcraze/discussions#1714] [Q&A]
Dear Bitcraze Support Team,

I hope this message finds you well.

Last summer, we purchased 20 Crazybolt 1.1 drones from Bitcraze. Most of them have been functioning excellently and have greatly contributed to our projects. However, we have encountered an issue with one of the units.

After experiencing multiple crashes, the affected drone now only displays the blue NRF indicator light and is unresponsive. In an attempt to resolve the issue, I used my STLink-V2 to reflash the following components:

1. NRF Bootloader
2. NRF Firmware
3. STM Bootloader
4. Crazyflie Firmware(via make cload)

After performing these steps, the fault occurred. I can connect to the drone normally using cfclient without any errors. However, the only issue is that the battery voltage measurement in cfclient shows 2.0V, whereas when I measure the battery with a multimeter, it reads 3.9V.

I believe that the incorrect battery voltage measurement does not cause any issues. Nevertheless, I proceeded to initiate take-off. The motors seem unable to reach sufficient RPM to make the drone take off (thrust shows 100%, but the sound is very low, and the RPM does not appear to reach the required levels). However, using the same motors, ESCs, and battery, I can successfully make another Bolt drone take off without any problems.

Additional Observations:

When measuring with a multimeter:
1. BATT is approximately 3.8V
2. VBAT is approximately 0.7V
These measurements are consistent on both the good and faulty Bolt drones.
Despite these efforts, the problematic drone remains non-functional. We have ensured that all connections are secure and followed the standard reflashing procedures, but the issue persists.

So I want to ask that,

1. Why is the battery voltage measurement faulty? What is the principle behind the battery voltage measurement in the Crazybolt 1.1?

2. Even if Bolt cannot sense the correct voltage, what impact does this have on the motors? Specifically, is there a relationship between the voltage displayed on cfclient (which is inaccurate) and the motor's control strategy?

Thank you for your prompt attention to this matter. We look forward to your support in resolving this issue as soon as possible.

Best regards,

SEU-NetSI

**Comment by ToveRumar** [comment]
Hi @mfxjx000205!

Thank you for providing such detailed description of your problem!

Due to holidays we are a bit low on staff at the moment, hence the delay.

1. I do not know exactly why this would be. Maybe @tobbeanton could give you a better answer here. 
2. By default there is a battery compensation running algorithm running in the Bolt FW. This can be turned of by setting ENABLE_THRUST_BAT_COMPENSATED to 0. I would suggest trying this to be able to further diagnose your issue. 

If this helps there is probably some damage to the on board power chip.
